### ISSUE TITLE
Fields with spaces don't work - should they?

### ISSUE THREAD

**Issue by ellisonbg** [vega/vega-lite#1460]
Please:
- [x] Check for duplicate issues
- [x] Describe how to reproduce the bug /  the goal of the new feature request
- [x] Provide an example spec and/or screenshots (if applicable) 

Field that have spaces in their names (common in Python Data Frames) don't work in vl specs. Here is an example:

```
{'data': {'values': [{'a b': 0.9228935641885049, 'c d': 0.5982771352733822},
   {'a b': 0.6819537235661846, 'c d': 0.4599852852378238},
   {'a b': 0.035533848836472814, 'c d': 0.6863884491167203},
   {'a b': 0.3491133898504284, 'c d': 0.7368153745410643},
   {'a b': 0.42475313594111874, 'c d': 0.8129620247637963},
   {'a b': 0.5239172250991592, 'c d': 0.5616266265239936},
   {'a b': 0.4668258532026234, 'c d': 0.436625706645942},
   {'a b': 0.2692059264553106, 'c d': 0.2746758424896977},
   {'a b': 0.6009825566170259, 'c d': 0.2648775327299746},
   {'a b': 0.7457746080337384, 'c d': 0.15435369511584507}]},
 'encoding': {'x': {'field': 'a b', 'type': 'quantitative'},
  'y': {'field': 'c d', 'type': 'quantitative'}},
 'mark': 'point'}
```

Should this work? If not, we should add a check in altair to not allow them. @jakevdp @domoritz

**Comment by kanitw** [comment]
@ellisonbg Thanks for reporting.  This is a known bug that we haven't succeeded fixing yet (#778). 
I think it is a good idea to not allow them in Altair for now.

**Comment by ellisonbg** [comment]
OK great!

On Sat, Jul 9, 2016 at 11:47 AM, Kanit Wongsuphasawat <
notifications@github.com> wrote:

> Closed #1460 <https://github.com/vega/vega-lite/issues/1460>.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/vega/vega-lite/issues/1460#event-717826831>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/AABr0DRQ-khdALqgd-utVPPKL2ESOvIwks5qT9CugaJpZM4JIpJm>
> .
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com
